lib/coverband.rb

```diff
@@ -34,13 +34,12 @@ class << self
   end
 
   def self.configure(file = nil)
-    configuration_file = file || CONFIG_FILE
-
+    configuration_file = file || ENV.fetch('COVERBAND_CONFIG', CONFIG_FILE)
     configuration
     if block_given?
       yield(configuration)
     elsif File.exist?(configuration_file)
-      require configuration_file
+      load configuration_file
```

Comment:
@danmayer I had to change this to load because when running all the tests at once, the configuration file may have already been loaded. If it had already been loaded, this method because essentially a noop and the configuration was not set again. This is a problem in an environment where we are running multiple tests scenarios with different configurations.

Reply:
makes sense